pandas/core/indexes/multi.py

```diff
@@ -1077,7 +1077,9 @@ def _set_names(self, names, level=None, validate=True):
             self.levels[l].rename(name, inplace=True)
 
     names = property(fset=_set_names, fget=_get_names,
-                     doc="Names of levels in MultiIndex")
+                     doc="""
+                     Names of levels in MultiIndex
+                     """)
```

Comment:
can you use `\n` instead of actual line breaks here?

Reply:
Not sure if i'm doing this right. Do let me know if this fixes it.